### ISSUE TITLE
Missing Spouse Report

### ISSUE THREAD

**Issue by txmusick** [ShammyLevva/FTAnalyzer#228]
I've utilized the report for spouses with the same surname, but I need to go a bit deeper. I'm looking for a report for daughters whose surname doesn't match their father's. I try to catch these as I pull in a census record, obituary, or findagrave, but I'm sure I have some lingering. I like to go into their record, add a spouse with the married surname, using a question mark for their first name, and then correct the daughter's surname to match their father. When I view my name index by given name first, those with a ? for a first name are at the top and I know to go research to fill in their info. Is there a report that catches daughters with a different surname than their father's? Would it be possible to get one? I love this tool, it's helping me clean up so much clutter and pinpoint errors. Thanks so much for creating it!

**Comment by SeaBorn49** [comment]
I agree!  I'm glad to see how helpful this would be!  Relationship reports can be easily constructed using Boolean logic.  A similar report to construct would be for childless couples!  That report could easily turn up numerous unknown relatives!
